src/clj/game/cards/agendas.clj

```diff
@@ -1722,12 +1722,6 @@
                                                              (wait-for (corp-install
                                                                          state side chosen-ice chosen-server
                                                                          {:ignore-all-cost true :index target})
-                                                                       (when (and run
```

Comment:
Any time you remove lines like this and you're left with `(wait-for (X) (effect-completed))`, you can condense it to a single `(X state side eid ...)` call.